x-pack/plugins/security_solution/public/common/components/events_viewer/index.tsx

```diff
@@ -220,6 +223,7 @@ type PropsFromRedux = ConnectedProps<typeof connector>;
 export const StatefulEventsViewer = connector(
   React.memo(
     StatefulEventsViewerComponent,
+    // eslint-disable-next-line complexity
```

Comment:
😢

Reply:
The complexity warning is a good warning that we can use to track tech debt. Looking below, this indeed looks close to needing a refactor from someone soon-ish.